sky/serve/replica_managers.py

```diff
@@ -819,6 +821,27 @@ def _handle_preemption(self, info: ReplicaInfo) -> bool:
     # ReplicaManager Daemon Threads #
     #################################
 
+    class ReplicaRecordAction(Enum):
+        KEEP = auto()
+        REMOVE_SCALE_DOWN = auto()
+        REMOVE_PREEMPTION = auto()
+        REMOVE_VERSION_MISMATCH = auto()
+
+    def get_replica_record_action(self,
```

Comment:
```suggestion
    def _get_replica_record_action(self,
```
nit: use underscore prefix for internal functions